workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowPage.tsx

```diff
@@ -66,10 +66,12 @@ export const WorkflowPage = () => {
             loading={loading}
             canRun={canRun}
           >
-            <WorkflowDetailsTabContent
-              loading={loading}
-              workflowOverviewDTO={workflowOverviewDTO?.data}
-            />
+            {workflowOverviewDTO && (
```

Comment:
actually this might not work well, I think when in loading state workflowOverviewDTO is undefined, and this way we won't show the loading state (skeletons) that we have

Reply:
maybe better pass it, and add a more simple loading handling, if loading show <Progress>
if workflowOverviewDTO not defined, show null
make these changes in WorkflowDetailsTabContent and not here